### PR TITLE
test:fix ddl ts and open tests 

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 

1. using `DoneTS` to sort DDL and DML. `DoneTS`: the commitTS of the Txn which sets the DDL job done.
2. when we pull DDL jobs from history list, cdc will query `DoneTS` from TiKV
3. skip DML if the table is truncated
4. change the ticking time in owner and processor.

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: cmd/client.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func init() {
 	cliCmd.Flags().StringVar(&pdAddress, "pd-addr", "localhost:2379", "address of PD")
 	cliCmd.Flags().Uint64Var(&startTs, "start-ts", 0, "start ts of changefeed")
 	cliCmd.Flags().Uint64Var(&startTs, "target-ts", 0, "target ts of changefeed")
-	cliCmd.Flags().StringVar(&sinkURI, "sink-uri", "root@tcp(127.0.0.1:3306)/test", "sink uri")
+	cliCmd.Flags().StringVar(&sinkURI, "sink-uri", "root@tcp(127.0.0.1:3306)/mysql", "sink uri")
```
[the remark this answers is not in the document]
Comment (leoppro) [reply cmd/client.go]: if we drop database `test`, the database connection will be disconnected

[the remark this answers is not in the document]
Comment (suzaku) [reply cmd/client.go]: Maybe leave `database` empty, just `/`?

[the remark this answers is not in the document]
Comment (leoppro) [reply cmd/client.go]: fixed

**Pull request merged**